INSTALL.md

```diff
@@ -0,0 +1,8 @@
+# Dependenices
```

Comment:
Can move the content in this file to README.md

Reply:
Don't have to move the content for now